kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -240,70 +241,71 @@ private static UInt32 convertIp4(String ipAddrString) throws UnknownHostExceptio
 
     private static String wifiModeConvert(String kuraMode) {
         switch (kuraMode) {
-        case "INFRA":
-            return "infrastructure";
-        case "MASTER":
-            return "ap";
-        default:
-            throw new IllegalArgumentException(String.format("Unsupported WiFi mode \"%s\"", kuraMode));
+            case "INFRA":
+                return "infrastructure";
+            case "MASTER":
+                return "ap";
+            default:
+                throw new IllegalArgumentException(String.format("Unsupported WiFi mode \"%s\"", kuraMode));
```

Comment:
I'm going to be a PITA here: I don't like mixing a `fix` commit with a `style` one. Furthermore there's the increased risk of merge conflicts.

Furthermore the Eclipse Clean Up profile doesn't like it...